WordPress/gutenberg
# Editor: Debounce autosave on last edit

This pull request seeks to change the behavior of autosave such that an auto save does not occur until after the delay from the last edit made. Thus, so long as a user continues typing, an autosave will never occur until they are finished typing.

**Testing instructions:**

Verify that upon making a change to e.g. the title or content, an autosave occurs after 10 seconds of no action. Verify the implications of this specific behavior in that repeatedly making changes before the 10 seconds interval will reset the timer.
